Update README.md

mciverza opened the pull request:
update deployment to override startup of "swiss-army-knife" pod to rather be "sleep 3600"

mciverza committed:
Update README.md

update deployment to override startup of "swiss-army-knife" pod to rather be "sleep 3600"

mciverza commented:
prevents immediate crashlooping

steflsd commented:
This is just in the README file?

Also - isnt sleep inifinity better.

mciverza commented:
it would be, yes

mciverza committed:
Update README.md

inifinity may be a better sleep

mciverza commented:
added the change to `infinity`
only README was changed

jgericke commented:
Thanks Mac

steflsd commented on README.md:
```diff
@@ -79,7 +79,7 @@ spec:
       containers:
       - args:
-        - "3600"
+        - "infinity"
```
Should be ['infinity']

steflsd reviewed (commented):
What happened to the name argument?

mciverza commented:
blegh. it was there originally. I was basing it on [kube docs](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/)
```
    command: ["printenv"]
    args: ["HOSTNAME", "KUBERNETES_PORT"]
```

but then used `oc get deployment/swiss-army-knife -o yaml`  and it didnt' have the square brackets anymore

mciverza committed:
follow official Kube docs. add square brackets

see  https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/

```
apiVersion: v1
kind: Pod
metadata:
  name: command-demo
  labels:
    purpose: demonstrate-command
spec:
  containers:
  - name: command-demo-container
    image: debian
    command: ["printenv"]
    args: ["HOSTNAME", "KUBERNETES_PORT"]
  restartPolicy: OnFailure
```

mciverza committed:
add `name`

mciverza commented:
now the name is back in. and the square brackets
and I tested that I could deploy it
